src/components/Scheme/SchemeContainer.tsx

```diff
@@ -132,15 +185,40 @@ const SubscribedSchemeContainer = withSubscription({
   checkForUpdate: ["schemeId"],
   createObservable: async (props: IProps) => {
     const arc = getArc();
-    const scheme = arc.scheme(props.schemeId);
+    const scheme = arc.scheme(props.schemeId) as any;
 
     // TODO: this may NOT be the best place to do this - we'd like to do this higher up
-
+    // why are we doing this for all schemes and not just the scheme we care about here?
```

Comment:
good point. Seems we should at a `scheme: scheme.id` to the where clause here.

Reply:
the only reason I can think of is that as it is done here it results in one subscription.  if filtered by scheme.id, then would be one subscription per scheme.   Which do you think is best, @jellegerbrandy ?